shared/lib_shared_inverter.cpp

```diff
@@ -230,7 +230,7 @@ void SharedInverter::calculateTempDerate(double V, double tempC, double& p_dc_ra
 
 double SharedInverter::getInverterDCMaxPower(double p_dc_rated)
 {
-    double inv_dc_max_power;
+    double inv_dc_max_power = p_dc_rated * util::kilowatt_to_watt;
```

Comment:
This line needs a comment (I propose "//if the inverter type isn't one of the following, assume that max power is equal to rated power". Also, I think with this line, the last "else if... ==NONE" statement & contents should be removed because it is redundant.

Reply:
Updated.